data.json

```diff
@@ -467,5 +467,57 @@
     "genres": ["comedy"],
     "type": "movie",
     "year": 1974
-  }
+  },
+  {
+   "title": "Grey's Anatomy",
+   "emojiImgs": "💊💉🧪⚕🍷",
+    "genres": [
+          "drama",
+          "romance"
+      ],
+      "type": "TV Series",
```

Comment:
Looks good, just noticed a couple of things that need to be updated! The types should be `tv`, `movie` or `musical`. Please change the ones listed as `TV Series` to `tv` instead. This field is going to be used for filtering so all TV entries need to match.